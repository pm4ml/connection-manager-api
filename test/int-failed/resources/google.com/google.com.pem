 0 s:/C=US/ST=California/L=Mountain View/O=Google LLC/CN=www.google.com
   i:/C=US/O=Google Trust Services/CN=Google Internet Authority G3
-----BEGIN CERTIFICATE-----
MIIEijCCA3KgAwIBAgIQDkCYvd2AsNM5Sg4Uh9d2XDANBgkqhkiG9w0BAQsFADBU
MQswCQYDVQQGEwJVUzEeMBwGA1UEChMVR29vZ2xlIFRydXN0IFNlcnZpY2VzMSUw
IwYDVQQDExxHb29nbGUgSW50ZXJuZXQgQXV0aG9yaXR5IEczMB4XDTE5MDMyNjEz
NDA0OVoXDTE5MDYxODEzMjQwMFowaDELMAkGA1UEBhMCVVMxEzARBgNVBAgMCkNh
bGlmb3JuaWExFjAUBgNVBAcMDU1vdW50YWluIFZpZXcxEzARBgNVBAoMCkdvb2ds
ZSBMTEMxFzAVBgNVBAMMDnd3dy5nb29nbGUuY29tMIIBIjANBgkqhkiG9w0BAQEF
AAOCAQ8AMIIBCgKCAQEAjTFp+4USvDIqcw1a4b743Ihut+UfUU0ompZU5Cl7Z17z
6sq3R8XuJWG3tveAKbIs0M2YhnYFqkGLelLRsZbfsReJUX5LLtDW1H0srCAfuCHs
j2hOEMuKZQQ9tAjGOIzSx2RmAEIn+3AWstobrczaG8UwKUZNRi4aA+bNX1gzGgqd
x637YuA76+5VWdFLeJeKuX7XVY8RfQI0tuSS9rwu3ZCL6LMiJ6Df0uTiyTU7IHCq
7OgyG95QXPxcRZNtH4dyRsNkrEtT+ySelJuwI5eZv746qbu6Bs/FW5VrXlA9gt/Y
42dE5U1Vfx5RkyUxBJ9FM+V8cSJPDRj73zHIalDBPwIDAQABo4IBQjCCAT4wEwYD
VR0lBAwwCgYIKwYBBQUHAwEwGQYDVR0RBBIwEIIOd3d3Lmdvb2dsZS5jb20waAYI
KwYBBQUHAQEEXDBaMC0GCCsGAQUFBzAChiFodHRwOi8vcGtpLmdvb2cvZ3NyMi9H
VFNHSUFHMy5jcnQwKQYIKwYBBQUHMAGGHWh0dHA6Ly9vY3NwLnBraS5nb29nL0dU
U0dJQUczMB0GA1UdDgQWBBTqQr2KplYFp0cGJXGf5Nsbd7KsuTAMBgNVHRMBAf8E
AjAAMB8GA1UdIwQYMBaAFHfCuFCaZ3Z2sS3ChtCDoH6mfrpLMCEGA1UdIAQaMBgw
DAYKKwYBBAHWeQIFAzAIBgZngQwBAgIwMQYDVR0fBCowKDAmoCSgIoYgaHR0cDov
L2NybC5wa2kuZ29vZy9HVFNHSUFHMy5jcmwwDQYJKoZIhvcNAQELBQADggEBAGZR
XYIPaJP/5bRtuIrX2kdkYdP5aegjsKmgPJNryqbIBKSxOYtEv/RuDNcedB3DU/Pl
w9YYSP+FtGs7jZCJCe9WqTJ2Kxm/83kuDHEeZvWnqQS3qBnmcrFoDJJooy7WfGMb
n5xXIcX0JHa/kvXvV5xpHIiuA9dFEtQt9jgQgP2r1DUGp+HU2iUAp3PA0OJLXATE
KNO84S+2Y5frOZy9k2/abzxl9muvR4TL/lcSp7NRybqVa6OoplgRjOdOVUvBMGz1
mTAdyIjuI1btu3y01bTRf2LfiES7hhmYGCZrjw8jIZyumRPNQhm+7B/l/3xX6Asy
pO0K1vKoryu9hbgqDtg=
-----END CERTIFICATE-----
