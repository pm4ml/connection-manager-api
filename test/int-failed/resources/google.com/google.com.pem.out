Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number:
            0e:40:98:bd:dd:80:b0:d3:39:4a:0e:14:87:d7:76:5c
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: C=US, O=Google Trust Services, CN=Google Internet Authority G3
        Validity
            Not Before: Mar 26 13:40:49 2019 GMT
            Not After : Jun 18 13:24:00 2019 GMT
        Subject: C=US, ST=California, L=Mountain View, O=Google LLC, CN=www.google.com
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (2048 bit)
                Modulus:
                    00:8d:31:69:fb:85:12:bc:32:2a:73:0d:5a:e1:be:
                    f8:dc:88:6e:b7:e5:1f:51:4d:28:9a:96:54:e4:29:
                    7b:67:5e:f3:ea:ca:b7:47:c5:ee:25:61:b7:b6:f7:
                    80:29:b2:2c:d0:cd:98:86:76:05:aa:41:8b:7a:52:
                    d1:b1:96:df:b1:17:89:51:7e:4b:2e:d0:d6:d4:7d:
                    2c:ac:20:1f:b8:21:ec:8f:68:4e:10:cb:8a:65:04:
                    3d:b4:08:c6:38:8c:d2:c7:64:66:00:42:27:fb:70:
                    16:b2:da:1b:ad:cc:da:1b:c5:30:29:46:4d:46:2e:
                    1a:03:e6:cd:5f:58:33:1a:0a:9d:c7:ad:fb:62:e0:
                    3b:eb:ee:55:59:d1:4b:78:97:8a:b9:7e:d7:55:8f:
                    11:7d:02:34:b6:e4:92:f6:bc:2e:dd:90:8b:e8:b3:
                    22:27:a0:df:d2:e4:e2:c9:35:3b:20:70:aa:ec:e8:
                    32:1b:de:50:5c:fc:5c:45:93:6d:1f:87:72:46:c3:
                    64:ac:4b:53:fb:24:9e:94:9b:b0:23:97:99:bf:be:
                    3a:a9:bb:ba:06:cf:c5:5b:95:6b:5e:50:3d:82:df:
                    d8:e3:67:44:e5:4d:55:7f:1e:51:93:25:31:04:9f:
                    45:33:e5:7c:71:22:4f:0d:18:fb:df:31:c8:6a:50:
                    c1:3f
                Exponent: 65537 (0x10001)
        X509v3 extensions:
            X509v3 Extended Key Usage: 
                TLS Web Server Authentication
            X509v3 Subject Alternative Name: 
                DNS:www.google.com
            Authority Information Access: 
                CA Issuers - URI:http://pki.goog/gsr2/GTSGIAG3.crt
                OCSP - URI:http://ocsp.pki.goog/GTSGIAG3

            X509v3 Subject Key Identifier: 
                EA:42:BD:8A:A6:56:05:A7:47:06:25:71:9F:E4:DB:1B:77:B2:AC:B9
            X509v3 Basic Constraints: critical
                CA:FALSE
            X509v3 Authority Key Identifier: 
                keyid:77:C2:B8:50:9A:67:76:76:B1:2D:C2:86:D0:83:A0:7E:A6:7E:BA:4B

            X509v3 Certificate Policies: 
                Policy: 1.3.6.1.4.1.11129.2.5.3
                Policy: 2.23.140.1.2.2

            X509v3 CRL Distribution Points: 

                Full Name:
                  URI:http://crl.pki.goog/GTSGIAG3.crl

    Signature Algorithm: sha256WithRSAEncryption
         66:51:5d:82:0f:68:93:ff:e5:b4:6d:b8:8a:d7:da:47:64:61:
         d3:f9:69:e8:23:b0:a9:a0:3c:93:6b:ca:a6:c8:04:a4:b1:39:
         8b:44:bf:f4:6e:0c:d7:1e:74:1d:c3:53:f3:e5:c3:d6:18:48:
         ff:85:b4:6b:3b:8d:90:89:09:ef:56:a9:32:76:2b:19:bf:f3:
         79:2e:0c:71:1e:66:f5:a7:a9:04:b7:a8:19:e6:72:b1:68:0c:
         92:68:a3:2e:d6:7c:63:1b:9f:9c:57:21:c5:f4:24:76:bf:92:
         f5:ef:57:9c:69:1c:88:ae:03:d7:45:12:d4:2d:f6:38:10:80:
         fd:ab:d4:35:06:a7:e1:d4:da:25:00:a7:73:c0:d0:e2:4b:5c:
         04:c4:28:d3:bc:e1:2f:b6:63:97:eb:39:9c:bd:93:6f:da:6f:
         3c:65:f6:6b:af:47:84:cb:fe:57:12:a7:b3:51:c9:ba:95:6b:
         a3:a8:a6:58:11:8c:e7:4e:55:4b:c1:30:6c:f5:99:30:1d:c8:
         88:ee:23:56:ed:bb:7c:b4:d5:b4:d1:7f:62:df:88:44:bb:86:
         19:98:18:26:6b:8f:0f:23:21:9c:ae:99:13:cd:42:19:be:ec:
         1f:e5:ff:7c:57:e8:0b:32:a4:ed:0a:d6:f2:a8:af:2b:bd:85:
         b8:2a:0e:d8
